speechbrain/utils/distributed.py

```diff
@@ -10,12 +10,55 @@
 import datetime
 import os
 from functools import wraps
+from typing import Optional
 
 import torch
 
 MAIN_PROC_ONLY: int = 0
 
 
+def rank_prefixed_message(message: str, rank: Optional[int]) -> str:
```

Comment:
I think you are right. I left it like that in case an user wants to perform a certain operation if `rank` is `None` but I think its better the way you are proposing.